src/Microsoft.Performance.SDK.Runtime/Discovery/AssemblyExtensionDiscovery.cs

```diff
@@ -452,22 +409,65 @@ public bool ProcessAssemblies(IEnumerable<string> directoryPaths, out ErrorInfo
             return true;
         }
 
+        private static HashSet<string> GetExclusionValues(string targetDirectory, ILogger logger)
+        {
+            Debug.Assert(!string.IsNullOrWhiteSpace(targetDirectory), $"{nameof(targetDirectory)} is null.");
+            Debug.Assert(Directory.Exists(targetDirectory), $"{nameof(targetDirectory)} is invalid.");
+
+            string configurationFile = Directory
+                .GetFiles(targetDirectory, AssemblyExtensionDiscovery.ExclusionsFilename)
+                .FirstOrDefault();
+
+            if (configurationFile != null)
+            {
+                try
+                {
+                    IEnumerable<string> lines = File
```

Comment:
Should we add a check to make sure lines do not contain directory separator characters? Maybe we should log a warning and ignore any entries that do